src/app/p/[projectId]/components/PayForm.tsx

```diff
@@ -19,16 +25,15 @@ export function PayForm({ projectId }: { projectId: bigint }) {
         string calldata memo,
         bytes calldata metadata
    */
-  const value = parseUnits("0.069", 18);
```

Comment:
shall we just keep this but use `value` instead? saves a line of duplication 🤷 

```
const payValue = value ? parseUnits(value, decimals) : 0n
```

Reply:
How would you go about changing

`<DecimalsInput value={value} onChange={setValue} size="lg"/>`

(since this has to be a string afaik)